_posts/2021-01-19-bazel-4-0.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: posts
+title: "Bazel 4.0"
+authors:
+  - communikit
+---
+
+Bazel 4.0 is a major release and Bazel’s first LTS release. It contains new features and backwards incompatible changes.
+
+## New features
+
+* Starlark-defined flags can now be shorthanded using `--flag_alias`.
```

Comment:
Julie provided the link to https://docs.bazel.build/versions/master/skylark/config.html#using-build-setting-aliases

Reply:
Added link.